javascript-arrays/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<head>
+  <title>Javascript Arrays</title>
+</head>
+<body>
+  <script src="main.js">
+    var colors= ['red','white','blue'];
+    console.log('value of colors[0]:',colors[0]);
+    console.log('value of colors[1]:',colors[1]);
+    console.log('value of colors[2]:',colors[2]);
+
+    var America= 'America is' + " "+ colors[0] + ", " + colors[1] + ", " + 'and'+ ' ' + colors[2];
+    console.log(America)
+
+    colors[2]= 'green';
+    var Mexico= 'Mexico is' + " " + colors[0] + ", " + colors[1] + ", " + 'and' + ' ' + colors[2];
+    console.log(Mexico)
+
+    console.log('value of colors:',colors);
+
+    var students= ['Pam', 'Zill', 'Vivian', 'Sarah']
+    var numberOfStudents= students.length
+    var classroom = 'There are' + ' ' + numberOfStudents + ' students in the class';
+    console.log(classroom);
+    var lastIndex= students.length - 1;
+    var lastStudent= lastIndex[2];
```

Comment:
We need to use `lastIndex` as an index in order to access last student value of `students`